Mathlib/Data/Fin/Basic.lean

```diff
@@ -530,6 +532,12 @@ theorem nontrivial_iff_two_le : Nontrivial (Fin n) ↔ 2 ≤ n := by
 -- porting note: here and in the next lemma, had to use `← Nat.one_eq_succ_zero`.
 #align fin.nontrivial_iff_two_le Fin.nontrivial_iff_two_le
 
+theorem bot_ne_top : (⊥ : Fin (n + 2)) ≠ ⊤ := _root_.bot_ne_top
```

Comment:
I'm not sure I see the point in this lemma, given the proof

Reply:
I think you are probably right. I think there probably is a use for `zero_ne_last` though (for `Fin.pred` and `Fin.castPred` purposes).

Reply:
I have removed this.